es-ds-styles/scss/_dropdown.scss

```diff
@@ -201,3 +201,137 @@
   padding: variables.$dropdown-item-padding-y variables.$dropdown-item-padding-x;
   color: variables.$dropdown-link-color;
 }
+
+// ==========================================
+// ES-DROPDOWN CUSTOM STYLES
+// ==========================================
+
+// Custom es-dropdown input styling
+.es-dropdown-input {
+  align-items: center;
+  background-color: variables.$input-bg;
+  background-clip: padding-box;
+  border: 1px solid variables.$gray-500;
+  border-radius: variables.$border-radius-xs;
+  color: variables.$input-color;
+  display: flex;
+  height: variables.$input-btn-height;
+  justify-content: space-between;
+  padding: 16px 16px 16px 32px;
+  font-family: variables.$input-font-family;
+  font-size: variables.$input-font-size;
+  font-weight: variables.$input-font-weight;
+  line-height: variables.$input-line-height;
+  text-decoration: none;
+  transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+  white-space: nowrap;
+  width: 325px;
+
+  &:hover, &:active {
+    border-color: variables.$blue-300;
+    .es-dropdown-panel {
+      border-top: 1px solid variables.$blue-300;
+    }
+  }
+
+  &:focus,
+  &:focus-visible {
+    border-color: variables.$gray-500;
+    outline: 0;
+    font-weight: variables.$font-weight-bold;
+  }
+
+  span[data-pc-section='input'] {
+    color: variables.$input-color-placeholder;
+    &:focus,
+    &:focus-visible {
+      outline: 0;
+      font-weight: variables.$font-weight-bold;
+    }
+  }
+
+  &:disabled,
+  &.disabled {
+    background-color: variables.$gray-50;
+    border-color: variables.$gray-500;
+    cursor: not-allowed;
+    span[data-pc-section='input'] {
+      color: variables.$gray-500;
+    }
+  }
+
+  &.es-dropdown-open {
+    border-bottom: none;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+}
+
+// Dropdown panel styling
+// Top corners have no radius so that it visually merges with the input
+.es-dropdown-panel {
+  background-color: variables.$white;
+  border: 1px solid variables.$gray-500;
```

Comment:
Nit: can we check with design about whether we want to provide any elevation to the dropdown's popup menu, in case of visual collision with another similar-looking input or dropdown, which, depending on how they're positioned, could cause confusion about where this dropdown ends and the other one begins?

<img width="423" height="472" alt="Screenshot 2025-07-30 at 1 41 41 PM" src="https://github.com/user-attachments/assets/0785bdc7-0f69-4b9e-969f-7ee34dcf34c0" />

One option could be to reuse the subtle shadow from the light variant of EsPopover.

<img width="313" height="332" alt="Screenshot 2025-07-30 at 1 43 35 PM" src="https://github.com/user-attachments/assets/6fbee877-621e-4806-972a-0ca814217d20" />